box2d/src/main/kotlin/ktx/box2d/worlds.kt

```diff
@@ -60,3 +61,69 @@ fun World.create(bodyDefinition: BodyDefinition): Body {
  * @see createWorld
  */
 val earthGravity = Vector2(0f, -9.8f)
+
+/**
+ * Callback lambda for ray-casts. This lambda is called when a ray-cast hits a fixture.
+ *
+ * The lambda accepts these parameters:
+ * - `[Fixture]`, the fixture hit by the ray
+ * - `[Vector2]`, the point of initial intersection
+ * - `[Vector2]`, the normal vector at the point of intersection
+ * - `[Float]`, the fraction of the distance from `start` to `end` that the intersection point is at
+ *
+ * The lambda should return:
+ * - `-1f`, ignore this fixture and continue
+ * - `0f`, terminate the ray cast
+ * - a fraction, clip the length of the ray to this point
+ * - `1f`, don't clip the ray and continue
+ *
+ * Can be used in place of [com.badlogic.gdx.physics.box2d.RayCastCallback] via Kotlin SAM conversion.
+ */
+typealias RayCastCallback = (Fixture, Vector2, Vector2, Float) -> Float
```

Comment:
While we're at it, the API can be improved by returning `RayCastAction` enum instead of a `Float`. This is something I really hated about the Box2D API. You wouldn't need to declare global constants then. Instead of simply passing the `RayCastCallback` instance to the `World`, you could wrap the callback in our custom adapter that converts the enum result into a `Float` to satisfy Box2D API needs. Remember to `inline` the `rayCast` methods if you decide to go with the enum and the wrappers.